syntax = "proto3";

package person;

service PersonService {
  rpc GetPersonById (PersonIdRequest) returns (PersonResponse);
}

message PersonIdRequest {
  int32 id = 1;
}

message PersonResponse {
  int32 id = 1;
  string first_name = 2;
  string last_name = 3;
  string company_name = 4;
}
